SmilesMiles/View/TaskListView.swift

```diff
@@ -8,52 +8,41 @@
 import SwiftUI
 
 struct TaskListView: View {
-    @State private var tasks: [Task] = []
-    @State private var newTask = ""
-    
+    @StateObject private var viewModel: TaskListViewModel
+
+    init(viewModel: TaskListViewModel = TaskListViewModel()) {
```

Comment:
Can you please check if we can create TaskViewModel's instance directly on line 11?
For example: @StateObject private var viewModel: TaskListViewModel()